### PR TITLE
keywriter: Update to latest version

### LINKED ISSUE #544: Update keywriter to latest version
Would be great to have this rm keywriter fork with smaller font size for edit mode:
https://github.com/danschrage/remarkable-keywriter

There have been few more people who wanted to have a writing experience with smaller font size.

### PR DESCRIPTION
**Pull request by matteodelabre**
* Reduce font size in edit mode
* Rotate the buffer using Ctrl-Left/Ctrl-Right in view mode
* Add Ctrl-Q shortcut to quit the app
* Restore cursor position when switching between modes

Fixes #544

### CODE REVIEW

**Review by LinusCDE — APPROVED** [review]
Tested on the rM 1. Works well!

Not sure if this is intentional, but the update also changed the app to be in portrait orientation.

Edit: Ah, yeah, with rotating the buffer this just seems to be the new default but can easily be changed with CTRL+Right Arrow as described.

CTRL+Q also works.

File: package/keywriter/package [hunk 1]
```diff
@@ -5,24 +5,24 @@
 pkgnames=(keywriter)
 pkgdesc="Markdown-enabled free writing app"
 url=https://github.com/dps/remarkable-keywriter
-pkgver=0.1.0-3
-timestamp=2019-07-13T06:27Z
+pkgver=0.2~20-gddc9e73-1
```
Comment (LinusCDE) [inline package/keywriter/package:8]:
```suggestion
pkgver=0.2~20-ddc9e732-1
```

I think the commit sha1 short has a type here.

**Review by LinusCDE — CHANGES_REQUESTED** [review]
Overlooked this. Seems there is a typo.

File: package/keywriter/package (hunk 1 above)
Comment (matteodelabre) [reply package/keywriter/package:8]: I’ll admit it’s a bit strange, but this is the format generated by running `git describe --tags` in the keywriter repo. Not sure why git adds the `g` prefix here.

Comment (LinusCDE) [reply package/keywriter/package:8]: Ah, cool, didn't even know this was [a thing](https://stackoverflow.com/questions/23939214/why-does-git-describe-prefix-the-commit-id-with-the-letter-g). Learned something new I guess.

**Review by LinusCDE — APPROVED** [review]

**Comment by matteodelabre** [comment]
Thanks for the review!

**Pull request merged**